### DISCUSSION TITLE
Close Command Palette with Esc

### DISCUSSION THREAD

**Discussion by alexisthual** [community/community#15193] [Other Feature Feedback, Questions, & Ideas]
Even though pressing `ctrl + k` again is ok, I'm used to hitting `Esc` to exit such searchlight bar :smile:
Thanks for the great feature preview anyways!

**Comment by LucasBrandt** [comment]
I would like to be able to close the command palette with `Esc` as well (when extensions do not prevent it). It feels natural to me, and I often try it automatically. Unfortunately, this brings my browser (Firefox on Mac) out of full-screen mode when I don't want to do that - it would be great to prevent the browser's default event at least.
